son-gtklic/requirements.txt

```diff
@@ -0,0 +1,8 @@
+Flask
```

Comment:
We need to have specific versions for each dependency, in order not to risk breaking anything.